test/open_mic_test.rb

```diff
@@ -0,0 +1,53 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/open_mic'
+require './lib/joke'
+require './lib/user'
+
+class OpenMicTest < Minitest::Test
+
+  def setup
+    @open_mic = OpenMic.new({location: "Comedy Works", date: "11-20-18"})
+    @sal = User.new("Sal")
+    @ali = User.new("Ali")
+    @joke_1 = Joke.new(1, "Why did the strawberry cross the road?", "Because his mother was in a jam.")
+    @joke_2 = Joke.new(2, "How do you keep a lion from charging?", "Take away its credit cards.")
+  end
+
+  def test_it_exists
+    # skip
+    assert_instance_of OpenMic, @open_mic
+  end
+
+  def test_it_has_attributes
+    # skip
+    assert_equal "Comedy Works", @open_mic.location
+    assert_equal "11-20-18", @open_mic.date
+    assert_equal [], @open_mic.performers
+  end
+
+  def test_it_can_welcome_performers
+    # skip
+    @open_mic.welcome(@sal)
+    @open_mic.welcome(@ali)
+
+    assert_equal [@sal, @ali], @open_mic.performers
+  end
+
+  def test_it_detects_no_jokes_repeated
+    skip
+    @ali.learn(@joke_1)
+    @ali.learn(@joke_2)
+
+    assert_equal false, @open_mic.repeated_jokes?
+  end
```

Comment:
good job here, remember that `refute` is the same as `assert_equal false, `, so `refute @open_mic.repeated_jokes?` is preferred